app/models/user.rb

```diff
@@ -144,7 +144,7 @@ def self.create_blocked_scope(blocked)
   validates_uniqueness_of :login, :if => Proc.new { |user| !user.login.blank? }, :case_sensitive => false
   validates_uniqueness_of :mail, :allow_blank => true, :case_sensitive => false
   # Login must contain letters, numbers, underscores only
-  validates_format_of :login, :with => /\A[a-z0-9_\-@\.]*\z/i
+  validates_format_of :login, :with => /\A[a-z0-9_\-@\.\s]*\z/i
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use the new "sexy" validations (`validates` ...).